pages/browse-our-faq/bitmark-protocol-faq.md

```diff
@@ -0,0 +1,59 @@
+
+* When data has provenance, information (processed data) can be authenticated without a trusted third party. 
+* End-to-end protocols are low-cost.
+* Having a global data provenance enhances local regulation (more below).
+
+## Why hasn’t this been done before?
+
+End-to-end protocols cannot rely on trusted third parties. In order to have such a protocol for information ownership/control we must have the ability to secure the provenance of data. Information is processed data. Before Bitcoin (specifically public blockchains), there was not a way to secure the provenance of anything digital without relying upon trusted third parties. 
+
+## How can information be controlled?
+
+The Bitmark protocol provides a set of rules governing how data can be held, used, and exchanged. This works because the data itself can be held locally or in the Cloud. The protocol provides safe, low-cost access control without requiring a trusted third party.
+
+When changes or ownership / control are needed, the underlying data can be transferred peer-to-peer, creating an unforgeable provenance. 
+
+## How can this protect digital rights? 
+
+Information is the key to human rights. And the key to controlling information is property rights. 
+
+Property rights are agreements of control across different trust boundaries. For example, you could have a property right to access a river running through your land. In the digital environment, a tech company could recognize a property right to use your data in their algorithm. 
+
+For most of human history, the dominant form of property has been personal property. The functionality of personal property has not usually depended on trusted third parties. Today, an increasing share of our personal property is digital, and our rights depend on BigTech. An end-to-end protocol for information control could record digital property rights and provide a better future. 
+
+Recognizing rights to the data can transform it into a more accessible and valuable resource. Apps and services can safely use this data to enrich people’s lives.
+
+## What about government regulation?
+
+An end-to-end protocol for securing the provenance of data is compatible and could help improve any local, government regulation by providing more transparency into the information. 
+
+## Why use a blockchain?
+
+Currently, the only technology we know that can secure the provenance of data without requiring a trusted third party is a public blockchain. 
+
+## How much does this cost?
+
+Right now it costs two times the bitcoin payment. We are working towards layer two options to radically lower this price.
+
```

Comment:
I don't understand what "two times the bitcoin payment" means. People will want more specific numbers.